bitnami/docker-entrypoint-initdb.d/install_timescaledb.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
We've uploaded the naming of these files to be prefixed with numbers in the main repo, can you update?

Reply:
Not sure I understand. Can you pls clarify what you are asking to be changed? thx

Reply:
If you look at the current master, this file is now been renamed to `000_install_timescaledb.sh`, so to mirror that, this file should be named that as well. A similar rename happened for the other file in this dir.

Reply:
Thx. Done